src/test/groovy/org/osiam/test/integration/RegistrationIT.groovy

```diff
@@ -129,6 +131,17 @@ class RegistrationIT extends AbstractIT {
         Extension extension = user.getExtension('urn:org.osiam:scim:extensions:addon-self-administration')
         extension.getField('activationToken', ExtensionFieldType.STRING) != null
         user.getGroups().get(0).getDisplay().equalsIgnoreCase("Test")
+
+        //Waiting at least 5 seconds to ensure that an email was send
+        Thread.sleep(5000)
```

Comment:
I know this is a bad idea to use `Thread.sleep` in a test... should I use [CountDownLatch](https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/CountDownLatch.html) instead? Wait until one email was received by greenmail, right?